src/CMakeLists.txt

```diff
@@ -91,6 +91,9 @@ foreach(PROG "asm" "fix" "gfx" "link")
                  ${rgb${PROG}_src}
                  ${common_src}
                  )
+  if(SUFFIX)
+    set_target_properties(rgb${PROG} PROPERTIES SUFFIX ${SUFFIX})
```

Comment:
Is this automatically picked up by CMake?

Reply:
What do you mean? `-DSUFFIX=...` is explicitly passed to the first `cmake` command (see the readme example).

Reply:
Yes, but what does it *do*?

Reply:
Oh; it's a [standard cmake property](https://cmake.org/cmake/help/v3.0/command/set_target_properties.html) that gets appended to the target name.

Reply:
Then be careful, as [it seems like it's meant for `.exe`](https://cmake.org/cmake/help/v3.0/prop_tgt/SUFFIX.html).

Reply:
Hmm. Given that it's manual anyway, I guess Windows users can pass `-DSUFFIX=-0.8.0.exe`? I'll note that in the readme.